### ISSUE TITLE
[Bug]: Help center icon incorrect placement on wp-admin

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#70923]
### Quick summary

Icon at wp-admin is placed to the left side of "Write" button, compared to Calypso pages:

<img width="307" alt="Screenshot 2022-12-08 at 09 41 02" src="https://user-images.githubusercontent.com/87168/206388100-7eb96e5d-765f-4ecd-af5a-103c77481e94.png">

Calypso:

<img width="339" alt="Screenshot 2022-12-08 at 09 40 33" src="https://user-images.githubusercontent.com/87168/206388104-2db379a6-5f21-44f2-a27c-9b93df2deb41.png">

### Steps to reproduce

1. Have an atomic site
2. Visit `/wp-admin/plugins.php`
3. See header, icon is to the left of Write button
4. Compare to calypso pages

Might happen on simple site wp-admin too, just didn't check.

### What you expected to happen

Consistency.

### What actually happened

👀 

### Platform (Simple, Atomic, or both?)

Atomic

### Reproducibility

Consistent

### Severity

None

### Available workarounds?

None

**Comment by simison** [comment]
FYI @alshakero to triage.

**Comment by alshakero** [comment]
Hi! I think it's impossible to move it more inside in wp-admin.

@arcangelini can elaborate because I'm not very clear on the details.

**Comment by simison** [comment]
Sounds weird because all those other icons are from us, not from core.

**Comment by arcangelini** [comment]
Yeah the challenge comes from Jetpack. The masterbar in WP-ADMIN is managed by Jetpack who removes everything that is set in `$wp_admin_bar` and then builds its own version using only the pieces you see there. During my research it was suggested the only way to survive that is by setting a crazy high priority on the insertion of the Help Center icon. The downside is that it outputs the icon out of order from the rest of the admin bar. [This is where it all happens](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L346-L398) in Jetpack, and [this is my crazy high number to overwrite](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php#L237)

It has been a little while since I worked on this so maybe there is something that I missed. Please let me know if you have any other ideas and I would be happy to give them a try!

**Comment by simison** [comment]
Could the HTML for the button get injected from that part of Jetpack code directly? Wrapped with `if ( $this->site_woa )` perhaps if that helps. Rest of the scripts could continue getting enqueued from ETK.

Alternatively adding a filter in Jetpack plugin, allowing you to filter and add the menu at right placement from ETK?

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple – Replicated
- Tested on Atomic – Replicated

📌 ACTIONS
- Assigned to Vertex
